lib/archethic/bootstrap.ex

```diff
@@ -212,10 +214,44 @@ defmodule Archethic.Bootstrap do
     Sync.publish_end_of_sync()
     SelfRepair.start_scheduler()
 
+    if P2P.authorized_node?() do
+      resync_network_chain([:oracle, :node_shared_secrets], P2P.authorized_and_available_nodes())
```

Comment:
This function should be called before sending end of sync notification I think

Reply:
Done!